app/controllers/content_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ContentController < ApplicationController
+  def index
+    @content = get_content
+  end
+
+private
+
+  def get_content
+    response = metrics_service.content_summary(date_range: date_range,
+      organisation: params[:organisation])
+    @content = ContentItemsPresenter.new(response.deep_symbolize_keys[:results])
+  end
+
+  def metrics_service
```

Comment:
I would break the ```MetricsService``` into services for the other two methods:
1. fetch series
2. fetch aggregations